JDTestTravel/src/main/java/com/symbio/test/JDTestTravel/helper/ElementUtility.java

```diff
@@ -64,25 +64,30 @@ public void clickDate(Date date) {
 			}
 
 			// If targeted year is larger, move to the next month
-			if (calendar.get(Calendar.YEAR) > Integer.parseInt(currentYearString)) {
+			if (calendar.get(Calendar.YEAR) > Integer
+					.parseInt(currentYearString)) {
 				nextMonthButton.click();
 			} else {
 				// Check target and actual month
-				String currentMonthString = currentDatePicker.findElement(By.cssSelector("table:nth-child(1) .currMo"))
-						.getText();
+				String currentMonthString = currentDatePicker.findElement(
+						By.cssSelector("table:nth-child(1) .currMo")).getText();
 
 				// If targeted month is larger, move to the next month
-				if (calendar.get(Calendar.MONTH) > Integer.parseInt(currentMonthString) - 1) {
+				if (calendar.get(Calendar.MONTH) > Integer
```

Comment:
Please remove this file from commit, which just re-formating, not changing anything.